pyat/at/tracking/particles.py

```diff
@@ -24,8 +25,10 @@ def _generate_sigma_matrix(ld0, emitx, emity, blength, espread, radiation):
             sig_matrix = emitx*ld0.R[0] + emity*ld0.R[1] + \
                          blength*espread*ld0.R[2]
         except AttributeError:
-            print('R matrices not found. (are you using linopt6?)')
-            print('Creating uncoupled sigma matrix')
+            warnings.warn(AtWarning('R matrices not found. '
```

Comment:
I radiation is off is this a problem? I R is not found can you compute it instead?